tests/gis_tests/distapp/tests.py

```diff
@@ -645,6 +645,19 @@ def test_distance_order_by(self):
         ).values_list('name', flat=True).filter(name__in=('San Antonio', 'Pearland'))
         self.assertSequenceEqual(qs, ['San Antonio', 'Pearland'])
 
+    @skipUnlessDBFeature("has_Distance_function")
+    def test_distance_not_deprecated(self):
+        if postgis:
+            hillsdale = AustraliaCity.objects.get(name='Hillsdale')
+            qs = AustraliaCity.objects.annotate(distance=Distance('point', hillsdale.point, spheroid=True))
+            [c.distance.m for c in qs]    # just evaluate
```

Comment:
Ah, yes!  ... and having `list(...)` without using the result makes it obvious (to me at least) that the whole purpose of `list()` is to force evaluation, so no comment required, or?